### PR TITLE
Force TSLv1.3 in WsDataConnection

### PR DESCRIPTION
**Pull request by pavel-kokolemin**

### CODE REVIEW

File: BlocksettleNetworkingLib/SslDataConnection.cpp [hunk 1]
```diff
@@ -70,7 +70,8 @@ bool SslDataConnection::openConnection(const std::string &host, const std::strin
    info.gid = -1;
    info.uid = -1;
    info.retry_and_idle_policy = bs::network::ws::defaultRetryAndIdlePolicy();
-   info.options = params_.useSsl ? LWS_SERVER_OPTION_DO_SSL_GLOBAL_INIT : 0;
+   info.options = 0;
+   info.options |= params_.useSsl ? LWS_SERVER_OPTION_DO_SSL_GLOBAL_INIT : 0;
```
Comment (Ation) [inline BlocksettleNetworkingLib/SslDataConnection.cpp:74]: I don't get the difference

Comment (pavel-kokolemin) [reply BlocksettleNetworkingLib/SslDataConnection.cpp:74]: Just using same format as in `SslDataConnection` to make sure both files are updated.

Comment (Ation) [reply BlocksettleNetworkingLib/SslDataConnection.cpp:74]: I mean that 
a = 0
a |= X;
is identical to
a = X;

Comment (Ation) [reply BlocksettleNetworkingLib/SslDataConnection.cpp:74]: so it just looks wrong. it should be either preserving existing flags or set new value. no need to set to 0 first

Comment (pavel-kokolemin) [reply BlocksettleNetworkingLib/SslDataConnection.cpp]: sure, fixed

**Pull request merged**